src/utils/updateTokenPrices.ts

```diff
@@ -92,6 +92,66 @@ export default function fetchTokenPricesUSD(
   )
 }
 
+export default function fetchTokenPricesUSD(
```

Comment:
`getTokenPrice` at the end of the file already fetches token prices by address/chain. It currently returns them by symbol, but changing `(tokens[address] as BasicToken).symbol` to `(tokens[address] as BasicToken).address` will do what you want. Prefer to use that since it doesn't rely on the consts file

Reply:
This api doesn't fetch prices for USX and potentially more coins.